polyglot.js

```diff
@@ -362,18 +380,86 @@ class PolyGlot {
         }
         sheet._polyglotEditor = true;
     }
-    renderJournalSheet(journalSheet) {
+
+    renderJournalSheet(journalSheet,html) {
         this._addPolyglotEditor(journalSheet);
-        if (journalSheet.entity.owner) return;
-        if (game.user.isGM && !game.settings.get("polyglot", "runifyGM")) return;
+        if (journalSheet.entity.owner || game.user.isGM) {
+            let runes = false;
+            let texts = [];
+            let openBtn = $(`<a class="polyglot-button" title="Toggle Runes"><i class="fas fa-unlink"></i> Runes</a>`);
+            openBtn.click(ev => {
+                let button = html.closest('.app').find('.polyglot-button')[0].firstChild
```

Comment:
You can use `ev.currentTarget.firstChild` instead

Reply:
Also do a `ev.preventDefault()` since it's an `<a>` tag.

Reply:
Done, but we should also prevent double click from minimizing the window, Im not really sure the best way to do it.